crates/resvg/Cargo.toml

```diff
@@ -22,7 +22,7 @@ pico-args = { version = "0.5", features = ["eq-separator"] }
 png = { version = "0.17", optional = true }
 rgb = "0.8"
 svgtypes = "0.15.0"
-tiny-skia = "0.11.4"
+tiny-skia = { version = "0.11.4", default-features = false, features = ["std", "simd"] }
```

Comment:
Those two are enabled by default. So I am not touching them. Just removing `png-format`.